### PR TITLE
Split components into smaller subcomponents and improved CSS slightly

### PR DESCRIPTION
**Pull request by DonatJR**
see #11

### CODE REVIEW

File: app/typescript/views/Card.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import Vue from 'vue';
+import Component from 'vue-class-component'
+import * as styles  from "../../styles/Card.css";
+
+@Component({
+    props: [
+        'content'
```
Comment (SailReal) [inline app/typescript/views/Card.ts]: `content` is unused or am I wrong?

**Review by SailReal — COMMENTED** [review]
The help page overwrites the formatting in the help answer...I think it was just an css tag to fix that...
screen shot 2018-01-15 at 10 42 41 am

But everything else looks extreeeeeeme nice :heart_eyes:

**Comment by DonatJR** [comment]
regarding the help page: this is more or less intentional, as keeping whitespace (the css tag you speak of that was used before) fucks up a websites layout beyond recognition if it is viewed on different screen sizes. the help needs to be formatted like the result but for that I first have to change the return value in the game controller => seperate issue?

**Comment by SailReal** [comment]
>  ...first have to change the return value in the game controller => seperate issue?

:+1: but I would say with milestone 1.0.x ...

**Review by SailReal — APPROVED** [review]

**Pull request merged**